spec/services/hyrax/custom_queries/navigators/find_files_spec.rb

```diff
@@ -22,78 +26,93 @@
     context 'when original file exists' do
-      let!(:original_file) { FactoryBot.create_using_test_adapter(:hyrax_file_metadata) }
-      let!(:fileset) { FactoryBot.create_using_test_adapter(:hyrax_file_set, files: [original_file], original_file: original_file) }
+      let(:original_file) { FactoryBot.valkyrie_create(:hyrax_file_metadata) }
+      let(:fileset) { FactoryBot.valkyrie_create(:hyrax_file_set, files: [original_file]) }
+
       it 'returns file metadata resource' do
         expect(query_handler.find_original_file(file_set: fileset).id.to_s).to eq original_file.id.to_s
       end
     end
 
     context 'when files do not exist' do
-      let!(:fileset) { FactoryBot.build(:hyrax_file_set) }
+      let(:fileset) { FactoryBot.build(:hyrax_file_set) }
+
       it 'raises error' do
         expect { query_handler.find_original_file(file_set: fileset) }
-          .to raise_error ::Valkyrie::Persistence::ObjectNotFoundError, "File set's original file is blank"
+          .to raise_error ::Valkyrie::Persistence::ObjectNotFoundError
       end
     end
 
     context 'when file_set does not respond to original file' do
-      let!(:fileset) { FactoryBot.build(:hyrax_resource) }
+      let(:fileset) { FactoryBot.build(:hyrax_resource) }
+
       it 'raises error' do
         expect { query_handler.find_original_file(file_set: fileset) }
-          .to raise_error ::Valkyrie::Persistence::ObjectNotFoundError, "Hyrax::Resource is not a `Hydra::FileSet` implementer"
+          .to raise_error ::Valkyrie::Persistence::ObjectNotFoundError
       end
     end
   end
 
   describe '#find_extracted_text' do
     context 'when extracted text exists' do
-      let!(:extracted_text) { FactoryBot.create_using_test_adapter(:hyrax_file_metadata) }
-      let!(:fileset) { FactoryBot.create_using_test_adapter(:hyrax_file_set, files: [extracted_text], extracted_text: extracted_text) }
+      let(:fileset) { FactoryBot.valkyrie_create(:hyrax_file_set, files: [extracted_text]) }
+
+      let(:extracted_text) do
+        FactoryBot.valkyrie_create(:hyrax_file_metadata, type: Hyrax::FileMetadata::Use::EXTRACTED_TEXT)
+      end
+
       it 'returns file metadata resource' do
         expect(query_handler.find_extracted_text(file_set: fileset).id.to_s).to eq extracted_text.id.to_s
       end
     end
 
     context 'when files do not exist' do
-      let!(:fileset) { FactoryBot.build(:hyrax_file_set) }
+      let(:fileset) { FactoryBot.build(:hyrax_file_set) }
+
       it 'raises error' do
         expect { query_handler.find_extracted_text(file_set: fileset) }
-          .to raise_error ::Valkyrie::Persistence::ObjectNotFoundError, "File set's extracted text is blank"
+          .to raise_error ::Valkyrie::Persistence::ObjectNotFoundError
       end
     end
 
-    context 'when file_set does not respond to extracted text' do
-      let!(:fileset) { FactoryBot.build(:hyrax_resource) }
+    context 'when file_set class does not does not have any files' do
```

Comment:
Did you mean "file_set class" or "file_set"? I see that as a bit confusing.

Reply:
uhhh yeah, this wording is awkward. i'll rephrase.